R/estimate_secondary.R

```diff
@@ -275,6 +290,7 @@ secondary_opts <- function(type = "incidence", ...) {
     )
   }
   data <- update_list(data, list(...))
+  attr(data, "class") <- c("secondary_opts", class(data))
```

Comment:
return tag in roxygen header needs to be updated.

Beyond this PR but this should probably live in `opts.R` for easier findability.

Reply:
I've updated the docs to mention the returned class.